src/libs/intents/localMethods.ts

```diff
@@ -193,6 +193,7 @@ interface CustomMethods {
   setGeolocationTrackingId: typeof setGeolocationTrackingId
   forceUploadGeolocationTrackingData: typeof forceUploadGeolocationTrackingData
   getDeviceInfo: typeof getDeviceInfo
+  isAvailable: typeof isAvailable
```

Comment:
It should not be needed to add this because isAvailable is typed in cozy-intent. Are you up to date with master in this branch?

Reply:
> It should not be needed to add this because isAvailable is typed in cozy-intent. Are you up to date with master in this branch?

that was probably the issue, I'm going to verify